verilog/analysis/verilog_linter.h

```diff
@@ -125,10 +127,18 @@ absl::Status PrintRuleInfo(std::ostream*,
                            absl::string_view);
 
 // Outputs the descriptions for every rule for the --help_rules flag.
-void GetLintRuleDescriptionsHelpFlag(std::ostream*, absl::string_view);
+// When custom citations are delivered substitute chosen rule description
+// with citations specified by user
+void GetLintRuleDescriptionsHelpFlag(std::ostream*, absl::string_view,
+                                     const CustomCitationMap&);
 
 // Outputs the descriptions for every rule, formatted for markdown.
-void GetLintRuleDescriptionsMarkdown(std::ostream*);
+// When custom citations are delivered substitute chosen rule description
+// with citations specified by user
+void GetLintRuleDescriptionsMarkdown(std::ostream*, const CustomCitationMap&);
+
+// Parse the file with custom citations
+CustomCitationMap ParseCitations(absl::string_view);
```

Comment:
The parameter should have a name and describe what it is. E.g. it is not quite clear from reading this if the parameter is the filename or the file content. It 'sounds' like the filename, but looking at the implementation, it is actually the content.
So naming it and adding things to the function header comment will help.